### PR TITLE
Kevin yoo/hmp 47 use shared page title component for h1 across pages

### PR DESCRIPTION
**Pull request by kevinyooky**

### CODE REVIEW

**Review by john-conroy — COMMENTED** [review]
Very minor change - I'd double check the figma mockup to make sure it's even necessary.

File: context/app/static/js/pages/Genes/Genes.jsx [hunk 1]
```diff
@@ -33,9 +34,7 @@ function Genes({ geneSymbol }) {
         <Typography variant="subtitle1" component="h2" color="primary">
           Gene
         </Typography>
-        <Typography variant="h1" component="h1">
-          {`${geneSymbol.toUpperCase()} (${geneCommonName})`}
-        </Typography>
+        <PageTitle>{`${geneSymbol.toUpperCase()} (${geneCommonName})`}</PageTitle>
```
[the remark this answers is not in the document]
Comment (kevinyooky) [reply context/app/static/js/pages/Genes/Genes.jsx:37]: Oh, I see what you're saying.

[the remark this answers is not in the document]
Comment (kevinyooky) [reply context/app/static/js/pages/Genes/Genes.jsx:37]: Double checked figma and it was actually supposed to be variant="h2". Current implementation is correct, but thanks for that note, that was a good point.

[the remark this answers is not in the document]
Comment (kevinyooky) [reply context/app/static/js/pages/Genes/Genes.jsx:37]: Checked again to see if I missed any instances in the codebase where I could apply your comment. I think I got them all.

**Review by john-conroy — APPROVED** [review]

**Pull request merged**